pom.xml

```diff
@@ -649,8 +613,8 @@
   </profiles>
 
   <properties>
-    <maven.compiler.source>1.6</maven.compiler.source>
-    <maven.compiler.target>1.6</maven.compiler.target>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
What about compile with Java 11 but with a release target 8？

Maybe that will cause some problems with no such method exception at runtime.

Reply:
That will need update ci too

Reply:
the docker should be using already 1.8?

Reply:
Yes it uses 1.8 already